members/C001056.yaml

```diff
@@ -0,0 +1,96 @@
+bioguide: C001056
+contact_form:
+  method: POST
+  steps:
+    - visit: "https://www.cornyn.senate.gov/node/5853"
+    - find:
+      - selector: input#edit-submitted-first-name
+    - fill_in:
+      - name: First Name
+        selector: input#edit-submitted-first-name
+        value: $NAME_FIRST
+        required: true
+      - name: Last Name
+        selector: input#edit-submitted-last-name
+        value: $NAME_LAST
+        required: true
+      - name: Email
+        selector: input#edit-submitted-email
+        value: $EMAIL
+        required: true
+      - name: phone
+        selector: "#edit-submitted-phone-number"
+        value: $PHONE
+        required: true
+      - name: Street
+        selector: "#edit-submitted-address-thoroughfare"
+        value: $ADDRESS_STREET
+        required: true
+      - name: City
+        selector: "#edit-submitted-address-locality"
+        value: $ADDRESS_CITY
+        required: true
+      - name: ZipCode
+        selector: "#edit-submitted-address-postal-code"
+        value: $ADDRESS_ZIP5
+        required: true
+      - name: message
+        selector: "#edit-submitted-write-a-brief-message"
+        value: $MESSAGE
+        required: true
+    - select:
+      - name: Topic
+        selector: "#edit-submitted-please-select-an-issue-youd-like-to-discuss"
+        value: $TOPIC
+        required: true
+        options:
+          Abortion: Abortion
+          Agriculture: Agriculture
+          Arts and Humanities: Arts and Humanities
+          Banking: Banking
+          Budget: Budget
+          Criminal Justice: Criminal Justice
+          Defense: Defense
+          Education: Education
+          Election Reform: Election Reform
+          Energy: Energy
+          Environment: Environment
+          Finance: Finance
+          Firearms: Firearms
+          Foreign Affairs: Foreign Affairs
+          Healthcare: Healthcare
+          Housing: Housing
+          Immigration: Immigration
+          Interior: Interior
+          Judiciary: Judiciary
+          Labor: Labor
+          Law Enforcement: Law Enforcement
+          Medicare: Medicare
+          Religion: Religion
+          Service Academies: Service Academies
+          Small Business: Small Business
+          Social Security: Social Security
+          Taxes/Economy: Taxes/Economy
+          Telecommunications: Telecommunications
+          Trade: Trade
+          Transportation: Transportation
+          Veterans: Veterans
+          Welfare: Welfare
+      - name: State
+        selector: "#edit-submitted-address-administrative-area"
+        value: Texas
+        required: true
+    - find:
+      - selector: "#edit-submitted-e-newsletter-opt-in-1"
+    - click_on:
+      - value: Newsletter opt out 
+        selector: "#edit-submitted-e-newsletter-opt-in-1"
+      - value: Submit
+        selector: input[type='submit'][value='Submit'] 
+    - find:
+      - selector: div.webform-confirmation
+    - find:
```

Comment:
Is this needed twice?

Reply:
Yes. Cornyn's form takes a bit of time to load after hitting submitting. Instead of the hard `wait` the yaml previously had, I'm adding two `find` steps. `find` steps have an ingrained "wait" of 5 secs to give them (potentially) enough time to try to find the element their supposed to find. If the first `find` times out after those 5s, the second should be able to find the selector.